streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -361,6 +363,25 @@ class StreamingContext private[streaming] (
 
   /**
    * Create a input stream that monitors a Hadoop-compatible filesystem
+   * for new files and reads them using the given key-value types and input format.
+   * Files must be written to the monitored directory by "moving" them from another
+   * location within the same file system. File names starting with . are ignored.
+   * @param directory HDFS directory to monitor for new file
+   * @param conf Hadoop configuration
+   * @tparam K Key type for reading HDFS file
+   * @tparam V Value type for reading HDFS file
+   * @tparam F Input format for reading HDFS file
+   */
+  def fileStream[
```

Comment:
Sorry, meant exposing them to `binaryRecordsStream`. That's calling `fileStream`, and to include `conf` as an arg we'll also need to either specify `filter` and `newFilesOnly` or pass defaults, right?